_data-prepper/pipelines/configuration/sinks/opensearch.md

```diff
@@ -31,9 +31,9 @@ pipeline:
       bulk_size: 4
 ```
 
-To configure an [Amazon OpenSearch Service](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html) sink, specify the domain endpoint as the `hosts` option:
+To configure an [Amazon OpenSearch Service](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html) sink, specify the domain endpoint as the `hosts` option and set `serverless` to `true`:
```

Comment:
Amazon OpenSearch Service (AOS) is different from Amazon OpenSearch Service Serverless (AOSS). I think we either want two sections (one for AOS and another for AOSS) or just keep it the way it was without any Serverless specific information.